### PR TITLE
Update authenticator hostname

### PR DESCRIPTION
**Pull request by albertkol**
Make `authenticator` follow same host structure as new `submit` and `find`. Also make generated cookie be shared across apps. At the moment cookie is created for `localhost:4004`, so `submit` or `find` does not share it.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -48,9 +48,10 @@ services:
     environment:
       - REDIS_INSTANCE_URI=redis://redis-data:6379
       - ACCOUNT_STORE_API_HOST=http://account-store:8080
-      - AUTHENTICATOR_HOST=http://localhost:4004
-      - POST_AWARD_FRONTEND_HOST=http://localhost:4002
-      - POST_AWARD_SUBMIT_HOST=http://localhost:4003
+      - AUTHENTICATOR_HOST=http://authenticator.levellingup.gov.localhost:4004
+      - COOKIE_DOMAIN=.levellingup.gov.localhost
+      - POST_AWARD_FRONTEND_HOST=http://find-monitoring-data.levellingup.gov.localhost:4001
+      - POST_AWARD_SUBMIT_HOST=http://submit-monitoring-data.levellingup.gov.localhost:4001
```
Comment (samuelhwilliams) [inline docker-compose.yml:54]: Think we can only merge this when the monolith stuff is merged - at the moment none (of the main branches) of the apps legitimately use these domains, I think?

They probably work though ... 🤷

Comment (albertkol) [reply docker-compose.yml:54]: Aye.. that's why I targeted `hack/monolith`.

Comment (albertkol) [reply docker-compose.yml]: Rebased and fixed conflicts.

Comment (samuelhwilliams) [reply docker-compose.yml]: rebased is one word for 'merged that branch into this one' 👀

**Comment by albertkol** [comment]
My mistake. I guess this never got merged..

**Pull request closed without merging**